prime-router/metadata/fhir_mapping/hl7/message/ORM_O01.yml

```diff
@@ -10,21 +10,41 @@ resources:
     segment: .PID
     group: PATIENT
     resourcePath: resource/Patient
+    isReferenced: true
     repeats: false
 
-  - resourceName: Observation
-    segment: .OBSERVATION.OBX
-    group: ORDER.ORDER_DETAIL.OBSERVATION
-    resourcePath: resource/Observation
+  - resourceName: ServiceRequest
+    segment: .ORC
+    group: ORDER
+    resourcePath: resource/ServiceRequest
     repeats: true
     isReferenced: true
     additionalSegments:
-      - OBX
-      - NTE
+      - .ORDER_DETAIL.OBR
+      - .ORDER_DETAIL.NTE
+      - PATIENT.PATIENT_VISIT.PV1
+      - MSH
+      - PATIENT.PID
 
-  - resourceName: ServiceRequest
-    segment: .ORC
+  - resourceName: Observation
+    segment: .ORDER_DETAIL.OBSERVATION.OBX
```

Comment:
Does this work? I thought the segment should be `.OBSERVATION.OBX`

Reply:
It does work.  `Observation.yml` supports extracting information from `ORC` and `OBR` in addition to `OBX`.  So, in order to pass in the `ORC` and `OBR`, we need to change the `group` to the higher level `ORDER` group which then requires the `segment` to have the longer qualified path.  It took us a while to figure out the correct combination of `group` and `segment` since some information in Caristix didn't align with the HAPI HL7v2 library.